pallets/dapp-staking-v3/src/migration.rs

```diff
@@ -59,10 +60,16 @@ mod v8 {
     use crate::migration::v7::TierParameters as TierParametersV7;
     use crate::migration::v7::TiersConfiguration as TiersConfigurationV7;
 
-    pub struct VersionMigrateV7ToV8<T, TierThresholds>(PhantomData<(T, TierThresholds)>);
+    pub struct VersionMigrateV7ToV8<T, TierThresholds, ThresholdVariationPercentage>(
+        PhantomData<(T, TierThresholds, ThresholdVariationPercentage)>,
+    );
 
-    impl<T: Config, TierThresholds: Get<[TierThreshold; 4]>> OnRuntimeUpgrade
-        for VersionMigrateV7ToV8<T, TierThresholds>
+    impl<
+            T: Config,
+            TierThresholds: Get<[TierThreshold; 4]>,
+            ThresholdVariationPercentage: Get<u32>,
```

Comment:
`Percent` would make more sense here (or `Permill`, `Perbill`).
That way you have guarantee that's it's not larger than 1 or 100%.